docs/usage.md

```diff
@@ -97,6 +97,22 @@ group-1,1,SPAdes,SPAdes-group-1.contigs.fasta.gz
 
 When supplying pre-computed assemblies, reads **must** also be provided in the CSV input format to `--input`, and should be the reads used to build the assemblies, i.e., adapter-removed, run-merged etc.. Preprocessing steps will not be ran on raw reads when pre-computed assemblies are supplied. As long reads are only used for assembly, any long read fastq files listed in the reads CSV are ignored.
 
+### Databases
+
+#### BUSCO
+
+BUSCO can download lineage datasets automatically as it needs them, but this process can be slow if the internet connection is unstable, and may lead to redundant downloads across different samples. To avoid this, you can provide a local BUSCO lineage dataset using the --busco_db parameter. The local directory must follow a specific structure for BUSCO to recognize it.
```

Comment:
```suggestion
BUSCO can download lineage datasets automatically as it needs them, but this process can be slow if the internet connection is unstable, and may lead to redundant downloads across different samples.
To avoid this, you can provide a local copy of a BUSCO lineage dataset using the --busco_db parameter.
The local directory must follow a specific structure for BUSCO to recognize it.
```